man/opt_common_subexpr.Rd

```diff
@@ -26,4 +26,12 @@ code <- paste(
   sep = "\n"
```

Comment:
there are several `man/` files that should not be included in this PR.

Reply:
Hmm, maybe the branch I created from your `develop` branch used an old `remote`. Whatever be the case, I'll look into it.